Compilation errors on openSUSE

**SLADE Version**:
3.1.1.1
**Operating System**:
Linux - openSUSE Tumblewed

**Issue Details**:
Basically a series of errors like ```/MapLine.h:132:11: error: could not convert '(const char*)"<line NULL>"' from 'const char*' to 'Debuggable'```

As I'm using the SUSE open build system to build SLADE (to rpm), you can see the full log [here](https://build.opensuse.org/package/live_build_log/home:luke_nukem/slade/openSUSE_Tumbleweed/x86_64)

Any help would be much appreciated as I don't have much time for troubleshooting myself.

--- a/src/Application/Main.h
+++ b/src/Application/Main.h
@@ -136,7 +136,7 @@ const string MAP_TYPE_NAMES[] = {
 };
 
 // Debug helper type
-#ifdef WXDEBUG
+#ifdef DEBUG
 #include <typeinfo>
 class Debuggable
 {
@@ -207,13 +207,14 @@ inline void LOG_DEBUG(
 }
 
 #define LOG_DEBUG_VAR(name) LOG_DEBUG(#name ": ", name)
-#else  // WXDEBUG
+#else  // DEBUG
 struct Debuggable {
-	string repr;
+	template<typename T>
+	Debuggable(T _unused) { }
 };
 #define LOG_DEBUG(...)
 #define LOG_DEBUG_VAR(name)
-#endif  // WXDEBUG
+#endif  // DEBUG
 
 
 

